engines/engine.go

```diff
@@ -127,16 +128,10 @@ type Capabilities struct {
 // compatibility when we add more optional methods to Engine.
 type EngineBase struct{}
 
-// PayloadSchema returns an empty CompositeSchema indicating that a nil
-// payload is sufficient.
-func (EngineBase) PayloadSchema() runtime.CompositeSchema {
-	return runtime.NewEmptyCompositeSchema()
-}
-
-// ConfigSchema returns an empty jsonschema indicating that no custom config is
-// required.
-func (EngineBase) ConfigSchema() runtime.CompositeSchema {
-	return runtime.NewEmptyCompositeSchema()
+// PayloadSchema returns an empty schematypes.Object indicating no contraints
+// on keys of the payload object.
+func (EngineBase) PayloadSchema() schematypes.Object {
+	return schematypes.Object{}
```

Comment:
Nope...`plugin.ConfigSchema` is not a thing `pluginProvider.ConfigSchema` is the schema for configuration required when creating a plugin...